scripts/travis_install.sh

```diff
@@ -26,12 +26,22 @@ function install_cc_env {
     pip install awscli
 }
 
-# install CodeClimate env in all conditions
-install_cc_env
+# Install black for initial formatting stage
+if [ "$1" == "format" ]; then
+    pip install black
```

Comment:
Since the python packages get cached, do we want to stick with the first version we install or should this be `pip install -U black` to check for upgrades each time? 

Note: I'm not entirely sure how this works, but I _think_ it will stay on the first version that gets cached, or is that not how this works?

Reply:
Hmm. On one hand, I'm wary of `black` getting updated and our CI suddenly no longer passing. On the other hand, it's currently beta software, so we probably don't want to fix the version number forever. 

I'll add a -U for now. If it ever breaks our CI, we'll then immediately fix the version number at whatever the last good version was. That way we'll at least know to fix the problems instead of picking one version and then forgetting to ever update it.